packages/mdc-theme/README.md

```diff
@@ -16,27 +16,33 @@ path: /catalog/theme/
   </a>
 </div>-->
 
-MDC Theme is a foundational module that provides theming to MDC-Web components, and also makes it available to
-developers as Sass functions and mixins, as CSS custom properties, and as a set of CSS classes.
+This color palette comprises primary and accent colors that can be used for illustration or to develop your brand colors.
```

Comment:
I took it straight from the Material Design guidelines page, so I'd like to leave it. I think it provides a link between the guidelines and are code.

And then I realized we didn't have a link to the guidelines! So I added that.